docs/src/analysis/sensitivity.md

```diff
@@ -697,6 +697,86 @@ Thus one should check the stability of the backsolve on their type of problem be
 enabling this method. Additionally, using checkpointing with backsolve can be a
 low memory way to stabilize it.
 
+## Sensitivity Analysis for chaotic systems (shadowing methods)
+
+Let us define the instantaneous objective ``g(u,p)`` which depends on the state `u`
+and the parameter `p` of the differential equation. Then, if the objective is a
+long-time average quantity
+```math
+\langle g \rangle_∞ = \lim_{T \rightarrow ∞} \langle g \rangle_T,
+```
+where
+```math
+\langle g \rangle_T = \frac{1}{T} \int_0^T g(u,s) \text{d}t,
+```
+under the assumption of ergodicity, ``\langle g \rangle_∞`` only depends on `p`.
+
+In the case of chaotic systems, the trajectories diverge with ``O(1)`` error and one
+finds that the average derivative is not the derivative of the average. "Traditional"
+forward and adjoint sensitivity methods also diverge because the tangent space
+diverges with a rate given by the Lyapunov exponent.
+
+Shadowing methods use renormalization to accurately compute derivatives w.r.t. the
+long-time average quantities. The following `sensealg` choices exist
+
+
+- `ForwardLSS(;alpha=CosWindowing(),ADKwargs...)`: An implementation of the forward
+  [least square shadowing](https://arxiv.org/abs/1204.0159) method. For `alpha`,
+  one can choose between two different windowing options, `CosWindowing` (default)
+  and `Cos2Windowing`, and `alpha::Number` which corresponds to the weight of the
+  time dilation term in `ForwardLSS`.
+- `AdjointLSS(;alpha=10.0,ADKwargs...)`: An implementation of the adjoint-mode
+  [least square shadowing](https://arxiv.org/abs/1204.0159) method. `alpha`
+  controls the weight of the time dilation term in `AdjointLSS`.
+- `NILSS(nseg, nstep; rng = Xorshifts.Xoroshiro128Plus(rand(UInt64)), ADKwargs...)`:  
+  An implementation of the [non-intrusive least squares shadowing](https://arxiv.org/abs/1611.00880)
+  method. `nseg` is the number of segments. `nstep` is the number of steps per
+  segment.
```

Comment:
mention these in the big list above as well, and mention that they are expanded in this section.